src/flb_hash.c

```diff
@@ -179,6 +179,34 @@ static void flb_hash_evict_random(struct flb_hash *ht)
     }
 }
 
+static void flb_hash_evict_less_used(struct flb_hash *ht)
+{
+    struct mk_list *tmp;
+    struct mk_list *head;
+    struct flb_hash_entry *entry;
+    struct flb_hash_entry *entry_less_used = NULL;
+
+    mk_list_foreach_safe(head, tmp, &ht->entries) {
```

Comment:
please use ```mk_list_foreach()``` only, the ```_safe``` variant is only needed if you are going to remove any entry from the list while iterating.

Reply:
Updated the code to use mk_list_foreach() instead of mk_list_foreach_safe.